packages/reactstrap-validation-select/AvResourceSelect.js

```diff
@@ -11,11 +11,15 @@ class AvResourceSelect extends Component {
     super(props);
     this.state = {};
     this.state.previousOptions = [];
+    this.state.dropdownHasBeenFocused = false;
   }
 
   select = createRef();
 
   loadOptions = (...args) => {
+    if (this.props.waitUntilFocused && !this.state.dropdownHasBeenFocused) {
```

Comment:
I think instead of adding this `requireSatisfied` logic by itself at the top you can put logic into that `boolean` `requiredSatisfied` just to clean up the flow of `loadOptions`

Reply:
good idea